pom.xml

```diff
@@ -37,7 +37,7 @@
     <folio-spring-support.version>8.1.2</folio-spring-support.version>
     <coffee-boots.version>4.0.0</coffee-boots.version>
     <folio-java-checkstyle.version>1.0.1</folio-java-checkstyle.version>
-    <applications-poc-tools.version>1.5.5-SNAPSHOT</applications-poc-tools.version>
+    <applications-poc-tools.version>1.5.6-SNAPSHOT</applications-poc-tools.version>
```

Comment:
increased version, because build failed with 1.5.5-SNAPSHOT, because there is no such artifact